roles/openshift_master_ca/vars/main.yml

```diff
@@ -4,3 +4,6 @@ openshift_master_ca_cert: "{{ openshift_master_config_dir }}/ca.crt"
 openshift_master_ca_key: "{{ openshift_master_config_dir }}/ca.key"
 openshift_master_ca_serial: "{{ openshift_master_config_dir }}/ca.serial.txt"
 openshift_version: "{{ openshift_pkg_version | default('') }}"
+
+openshift_docker_image_name: openshift/origin
+openshift_docker_image: "{{ openshift_docker_image_name }}:{{ openshift_pkg_version | default('latest') }}"
```

Comment:
Fixed, now using openshift.common.cli_image. The way admin_binary has been abstracted it isn't really necessary but I'd rather making pulling the image explicit so it's more clear why it failed when/if it fails to generate the certs.